src/lib/utils.ts

```diff
@@ -97,3 +97,30 @@ export const contentTypes: Record<string, string> = {
 }
 
 export const imageTypes = ['.png', '.jpg', '.jpeg', '.gif', '.svg', '.tiff', '.webp']
+
+/**
+ * Robust stringification of any value, including json and bigints.
+ */
+export function stringify(value: unknown): string {
```

Comment:
Why is this `stringify` different than the default one in HighTable? Looks like the difference is indentation? Is there a reason indentation _shouldn't_ be the default in hightable?

Reply:
I'll update hightable to use the same code and upgrade here later.

Reply:
Done